.circleci/src/jobs/build_android.yml

```diff
@@ -13,6 +13,9 @@ working_directory: ~/project/frontend
 steps:
   - prepare_workspace
   - install_app_toolbelt
+# uses the ruby orb and install minimum ruby version required by fastlane
```

Comment:
🙃 I think this is a little redundant as the command is quite self explanatory imo.
```suggestion
```